reactioncommerce/reaction
# [WIP] Make some search results clickable

Resolves #2625.

This PR fixes the bug reported in #2625. In that bug, clicking on search results for accounts or orders did nothing, as opposed to clicking on the search result for products, which brought up that product's detail page. This PR now shows a dashboard/details page for any accounts/orders search result when it is clicked.

## Test instructions

* Login as admin.

* Perform a search.

* Open the `ACCOUNTS` tab of the search results. Click any of the search results, and you'll notice that it now shows the accounts dashboard.

* Open the `ORDERS` tab of the search results. Click any of the search results, and you'll notice that it now shows the orders dashboard.
